source/connect/mongoclient.txt

```diff
@@ -0,0 +1,132 @@
+
+- A **connection URI**, also known as a *connection string*, which tells the {+driver-short+}
+  which MongoDB deployment to connect to.
+- A **MongoClient** object, which creates the connection to and performs 
+  operations on the MongoDB deployment.
+
+You can also use ``MongoClientSettings`` to customize the way the {+driver-short+} behaves
+while connected to MongoDB.
+
+This guide shows you how to create a connection string and use a ``MongoClient`` object
+to connect to MongoDB.
+
+.. _scala-connection-uri:
+
+Connection URI
+--------------
+
+A standard connection string includes the following components:
+
+.. TODO, add this as last sentence for ``username:password`` description once a scala auth page is made: 
+.. For more information about the ``authSource`` connection option, see :ref:`scala-sync-auth`. 
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Component
+     - Description
+
+   * - ``mongodb://``
+
+     - Required. A prefix that identifies this as a string in the
+       standard connection format.
+
+   * - ``username:password``
+
+     - Optional. Authentication credentials. If you include these, the client
+       authenticates the user against the database specified in ``authSource``.
+
+   * - ``host[:port]``
+
+     - Required. The host and optional port number where MongoDB is running. If you don't
+       include the port number, the driver uses the default port, ``27017``.
+
+   * - ``/defaultauthdb``
+     
+     - Optional. The authentication database to use if the
+       connection string includes ``username:password@``
+       authentication credentials but not the ``authSource`` option. If you don't include 
+       this component, the client authenticates the user against the ``admin`` database.
+
+   * - ``?<options>``
+
+     - Optional. A query string that specifies connection-specific
+       options as ``<name>=<value>`` pairs. See
+       :ref:`scala-connection-options` for a full description of
+       these options.
+
+For more information about creating a connection string, see
+:manual:`Connection Strings </reference/connection-string>` in the
+MongoDB Server documentation.
+
+Atlas Connection Example
+------------------------
+
+To connect to a MongoDB deployment on Atlas, you must first create a client. 
+
+You can pass a connection URI as a string to the ``MongoClient.create()`` method 
+to connect to a MongoDB instance:
+
+.. literalinclude:: /includes/connect/ClientBasic.scala
+   :start-after: start-create-a-client
+   :end-before: end-create-a-client
+   :language: scala
+   :copyable:
+   :dedent:
+
+You can also create a client with your desired configurations by passing a 
+``MongoClientSettings`` object to a ``MongoClient`` object.
+
+To instantiate a ``MongoClientSettings`` object, use the builder method to 
+specify your connection string, using the ``applyConnectionString()`` method, 
+and any other client options. Once you have your desired configuration, 
```

Comment:
The way this is worded, it seems like you pass the connection string into `builder()`, but in the code sample it's passed into `applyConnectionString()`. I think this rephrasing will be a little more clear?
```suggestion
To instantiate a ``MongoClientSettings`` object, call the ``builder()`` method, then
chain the ``applyConnectionString()`` method and pass in your connection string. You can
also use the ``builder()`` method to specify any other client options. Once you have your desired configuration, 
```